recipes/embree3/all/conanfile.py

```diff
@@ -0,0 +1,153 @@
+
+import os
+from conans import CMake, tools
+from conans.tools import Version
+from conans.model.conan_file import ConanFile
+from conans.errors import ConanInvalidConfiguration
+import glob
+
+
+class Embree(ConanFile):
+    name = "embree3"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake"
```

`cmake_find_package` is useless if there is no requirement